llvm/lib/CodeGen/StackMaps.cpp

```diff
@@ -857,7 +843,13 @@ void StackMaps::emitCallsiteEntries(MCStreamer &OS) {
         OS.emitIntValue(0, 1); // Reserved
         OS.emitInt16(Loc.Size);
         OS.emitInt16(Loc.Reg);
-        OS.emitInt16(Loc.ExtraReg); // Reserved
+        OS.emitInt16(0); // Reserved
+        if (YkStackMapAdditionalLocs) {
+          OS.emitInt16(Loc.Extras.size());
```

Comment:
What happens if `Extra.size()` exceeds 16 bits? [I know it's unlikely but ....]

Reply:
`emitInt16` has asserts that check for this.